media/css/firefox/quantum-landing.scss

```diff
@@ -0,0 +1,805 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+@import '../hubs/sections';
+
+$color-quantum-blue: #0a84ff;
+
+/* -------------------------------------------------------------------------- */
+// Common elements
+
+body {
+    @include open-sans;
+}
+
+#outer-wrapper {
+    overflow-x: hidden;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-weight: normal;
+}
+
+a.download-link.button.button-green {
```

Comment:
Rather than overriding styles, it would be nice here to make a new button style and `@import` it into the stylesheet. Similar to what we've done [here](https://github.com/alexgibson/bedrock/blob/bug-1397381-quantum-download-page-redesign/media/css/firefox/new/quantum/_buttons.scss#L10-L31). Give it a button name like `quantum`. This will make it easier for us to consolidate styles between pages in the future.

Reply:
So, the reason I did the override is because I am using the `download_firefox` helper. Now, although you cannot specify a custom class, I see there is a parameter called `button_color` with a default value of `button-green` so, I guess I can use that as a way to specify a custom class, say

`{{ download_firefox(alt_copy=_('Download Now'), dom_id='firefox-desktop-page', download_location='other', button_color='quantum-download') }}`